operator/config/crd/bases/forklift.konveyor.io_migrations.yaml

```diff
@@ -380,6 +380,9 @@ spec:
                     operatingSystem:
                       description: The Operating System detected by virt-v2v.
                       type: string
+                    newName:
+                      description: The new name of the VM after matching DNS1123 requirements.
+                      type: string
```

Comment:
did you automatically generated it? seems CI fails on this.

Reply:
yes it automatically generated, I"ll look into it